path_revoke.go

```diff
@@ -1,60 +1,60 @@
 package keyfactor
 
-import (
```

Comment:
I think this functionality was moved to `path_cert.go` should we just delete this file?